vector/src/main/AndroidManifest.xml

```diff
@@ -242,7 +243,7 @@
                 <data
                     android:host="room"
                     android:scheme="element" />
-
+                <data android:scheme="schildichat" />
```

Comment:
Why not restrict to `room` and `user` like upstream does? Furthermore, this likely conflicts with the one above since you haven't restricted the SSO one either?